javatests/arcs/core/tools/Proto2SchemaTest.kt

```diff
@@ -0,0 +1,54 @@
+package arcs.core.tools
+
+import arcs.core.data.FieldType
+import arcs.core.data.Schema
+import arcs.core.data.SchemaFields
+import arcs.core.data.SchemaName
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+
+@RunWith(JUnit4::class)
+class Proto2SchemaTest {
+
+    @Test
+    fun schemaGeneration_singleProperty() {
```

Comment:
Shouldn't a test for proto2schema take a proto and produce a schema instead?

Reply:
Yes, totally. Here, I'm testing an internal method, since I was waining on a few things (e.g. the schema protos)  I am going to have a discussion with Ray today to discuss how the output Kt file should be structured.